server/routes/conversation.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const passport = require("passport");
+const Conversation = require("../database/models/conversation");
+const User = require("../database/models/user");
+const router = express.Router();
+
+router.get("/user/:id/conversations",
```

Comment:
I had missed this earlier. Same comment as the other PR. Remove /conversations. This endpoint above due to routes setup will end up looking like:
`/conversations/user/:id/conversations`
After removing /conversations from the end above endpoint looks like:
`/conversations/user/:id`

Reply:
done